tests/utils/session.py

```diff
@@ -86,12 +86,12 @@ def checked_delete(self, path: str, **kwargs) -> dict:
         self.calls.append(FakeCall('DELETE', path, params=kwargs.get('params')))
         return self._get_response()
 
-    def _get_response(self):
+    def _get_response(self, default_response: dict = {}):
```

Comment:
python has a thing about default mutable arguments: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Fixed this (used the pattern in the link)